README.md

```diff
@@ -63,7 +63,14 @@ To run **Sugarizer Application from the Web Browser** (GNU Linux/Mac OS/Windows)
 
 For **Chrome**, close ALL running instances of Chrome and re-launch it using the command line:
 
-    chrome --allow-file-access-from-files index.html
+ 	chrome --allow-file-access-from-files  index.html
+
+On Linux, you should launch:
+
+	google-chrome-stable --allow-file-access-from-files index.html
+
+> Note: you can get chrome-package-name by running `sudo dpkg -l | grep chrome`, which is `google-chrome-stable` in Ubuntu
```

Comment:
It will be better to say that 'google-chrome-stable' is the name of Chrome in Ubuntu but could be different on other distribution, you can get...